lansa-vmss-windows-autoscale-sql-database/mainTemplate.json

```diff
@@ -653,6 +654,30 @@
             }
          }
       },
+    {  
+      "apiVersion": "2019-11-01",
+      "type": "Microsoft.Network/networkSecurityGroups",
+      "name": "[variables('frontEndNSGName')]",
+      "location": "[parameters('location')]",
+       "properties": {
+         "securityRules": [
+           {
+            "name": "rdp-rule",
+            "properties": {
+            "description": "Allow RDP",
+            "protocol": "Tcp",
+            "sourcePortRange": "*",
```

Comment:
This SourcePort range must be the range of the NAT rules : natDBStartPort to natEndPort